### ISSUE TITLE
SharedArrayBuffer warning

### ISSUE THREAD

**Issue by shin0602** [diegomura/react-pdf#1343]
Is it possible to update the dependency react-reconciler to remove the SharedArrayBuffer warning?

packages/renderer/package.json

```json
-    "react-reconciler": "^0.23.0",
+    "react-reconciler": "^0.26.2",
```

**Comment by diegomura** [comment]
I don't think that would be possible without updating react peer dependency as well, and that's a bit blocked for the moment (see https://github.com/diegomura/react-pdf/issues/1062). I'll leave that as reference for these issues. Thanks!
